### DISCUSSION TITLE
Gradle version check and refresh?

### DISCUSSION THREAD

**Discussion by Arc-E-Tect** [Splitties/refreshVersions#595] [Q&A] [answered]
Hi,

Is there a way to use the plugin to check whether there is a new version of Gradle available?

Before using the refreshVersions plugin, I used the dependencyUpdates plugin by Ben Manes, which performs this check. I found it very convenient to stay up to date with Gradle.

Thanks

**Comment by jmfayard** [comment, marked as answer]
The plugin used to be just an extension of Ben Manes plugin so I'm well aware of this feature.

If you run `./gradle refreshVersions` it will be displayed in the console.

I agree it's easy to miss it

**Reply by Arc-E-Tect** [reply]
Ah yes. The blue line saying `modified:   versions.properties` got all my attention and made me miss the comment that there is a new Gradle version.

Thanks
